src/global/reco/reco.cc

```diff
@@ -37,7 +36,6 @@ void InitPlugin(JApplication *app) {
     app->Add(new JChainMultifactoryGeneratorT<MatchClusters_factory>(
         "ReconstructedParticlesWithAssoc",
         { "EcalEndcapNClusters",
-          "EcalBarrelScFiClusters",
```

Comment:
This doesn't seem right...

Reply:
😩 Aggh. I deleted that by accident. Thank you for catching that!